src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -63,6 +64,102 @@ public function iSortShippingMethodsByName(string $sortType = 'ascending'): void
         ]);
     }
 
+    /**
+     * @When I add the "Total weight less than or equal" rule configured with :weight
```

Comment:
I feel like `Total weight less/greater than or equal` could parametrized and stringified into the type, might reduce the number of functions a bit.